### DISCUSSION TITLE
HD HUD in different langages - Sharing the source file for the HUD ?

### DISCUSSION THREAD

**Discussion by BerengerBrillante** [leoetlino/project-restoration#257] [General]
Hello,

I'm currently preparing my Odin 2 for multiple Mods on various consoles.
I'd like my kids to play with every Zelda games I have. 

I like this restoration project as a whole, but I would love to be able to have my kids read their own langage (French)
I'd like to know if the modder could share the source file (like a psd file ?) for the HD HUD ?
It would be easier than manually editing the tex file.

I would translate it to french, and send it back here for everyone to enjoy it with pleasure.

Thanks,

**Comment by leoetlino** [comment]
Sorry, I don't have the source files for the textures either! Consider joining this Discord server, which is where I grabbed the HD HUD textures from: https://discord.gg/PT8qyWh6GE
